includes/themes.php

```diff
@@ -964,6 +969,18 @@ public function install_theme( $file ) {
 	public function sync_settings() {
 		_deprecated_function( __METHOD__, '5.0' );
 	}
+
+	/**
+	 * Shows admin notice to update the plugin.
+	 *
+	 * @returns {void}
+	 */
+	public function update_bd_admin_notice() {
+		$class = 'notice notice-warning';
+		$message = __( 'Please update business directory plugin to latest version.', 'business-directory-plugin' );
+
+		printf( '<div class="%1$s"><p>%2$s</p></div>', esc_attr( $class ), esc_html( $message ) );
```

Comment:
Instead of printing the notice directly, let's use the message functions that already exist. (wpbdp_admin_message)